tests/sles4sap/trento/test_hana_unregister_node.pm

```diff
@@ -0,0 +1,82 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Trento unregister the secondary node
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use base 'consoletest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils 'script_retry';
+use qesapdeployment;
+use trento;
+
+sub run {
+    my ($self) = @_;
+    select_serial_terminal;
+
+    my $secondary_host = 'vmhana02';
+    cluster_print_cluster_status($secondary_host);
+
+    # Set secondary node in maintenance mode
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+    qesap_ansible_cmd(cmd => "sudo crm configure property maintenance-mode=true",
+        provider => $prov,
+        filter => $secondary_host);    
+
+    # Stop hana instance
+    $cmd_stop = join(' ', 'HDB', 'stop');
+    cluster_hdbadm($secondary_host, $cmd_stop);
```

Comment:
I think that for a such simple command you can use
```
cluster_hdbadm($secondary_host, 'HDB stop');
```

Reply:
Is it not some time needed here to let Trento web to keep up with the stop state?

Reply:
wait applied